build.gradle

```diff
@@ -12,10 +12,16 @@ allprojects {
         implementation group: 'com.fasterxml.jackson.core', name: 'jackson-annotations', version: jacksonVersion
         implementation group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: jacksonVersion
         implementation group: 'org.apache.commons', name: 'commons-lang3', version: apacheCommon
+
+        //Dependency to converter Duration to string and vice versa

```

Comment:
I'm not sure that hibernate-core is for only Duration conversion :)

Reply:
tried to remove this dependency but I need to use these imports:  `import org.hibernate.annotations.CreationTimestamp;
import org.hibernate.annotations.UpdateTimestamp;`
for "create_date" and "last_update_date" to work properly